challenge-number.json

```diff
@@ -1,6 +1,6 @@
 {
   "total": 45,
```

Comment:
update total to 46.

Reply:
I need to create a new schematic to update a challenge to another number.